include/Core/Posts/Modules/DeletePostsByTaxonomyModule.php

```diff
@@ -12,6 +12,9 @@
  * @since 6.0.0
  */
 class DeletePostsByTaxonomyModule extends PostsModule {
+	/**
```

Comment:
@rajanit2000 

Why is this file changed?

I am seeing lot of whitespace changes, so I am not sure what changes you made to this file.

Reply:
Its update when run phpcbf command, How can i revert it?